Project Health Check/MainView.swift

```diff
@@ -2,17 +2,12 @@
 //  ContentView.swift
 //  Project Health Check
 //
-//  Created with ❤️‍🔥 by Michael Caesario on 15/12/23.
-//  Copyright © 2023 Michael Caesario. All rights reserved.
-// 
-//  Website: https://mikecaesario.app
-//  GitHub: https://github.com/mikecaesario
-//  LinkedIn: https://www.linkedin.com/in/mikecaesario/
+//  Created by Darius, James, and Michael on 15/12/23.
 //
 
 import SwiftUI
 
-struct ContentView: View {
+struct MainView: View {
```

Comment:
Mike I just leave a comment just to be sure, because I don.t know SwiftUI that good. Here shouldn.t be the tab bar with the 3 tabs of the views you created ?